scripts/agency_ingestion/pyproject.toml

```diff
@@ -10,7 +10,7 @@ boto3 = "^1.24.69"
 requests = "^2.28.1"
 bs4 = "^0.0.1"
 click = "^8.1.3"
-pymongo = "^4.2.0"
+pymongo = {extras = ["srv"], version = "^4.2.0"}
```

Comment:
I actually pushed dns python to main for agency ingestion and data update, so these files can be reverted.